frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -185,8 +185,15 @@ export interface DetailsGridProps
 }
 
 export function DetailsGrid(props: DetailsGridProps) {
-  const { sectionsFunc, resourceType, name, namespace, children, ...otherMainInfoSectionProps } =
-    props;
+  const {
+    sectionsFunc,
+    resourceType,
+    name,
+    namespace,
+    children,
+    actions,
```

Comment:
You don't need the `actions` here because they're already included in the `...otherMainInfoSectionProps`.

Reply:
updated